src/cmdline.rs

```diff
@@ -341,6 +427,12 @@ pub fn parse_args() -> Result<Config> {
             ("remove", Some(matches)) => parse_iso_remove(&matches),
             _ => bail!("unrecognized 'iso' subcommand"),
         },
+        ("osmet", Some(osmet_matches)) => match osmet_matches.subcommand() {
```

Comment:
Longer term, how do you plan to arrange the CLI? My understanding is that here in `coreos-installer` we will want to expose only the porcelain entrypoint on top of the current `unpack` logic, right? If so, where does the mapping/packing belong?

Reply:
See the last paragraph of https://github.com/coreos/coreos-installer/pull/187#issue-387995035. We could mark the `osmet` command as hidden in the end.

Packing happens in cosa: https://github.com/coreos/coreos-assembler/pull/1244.

Reply:
Ack.
I think I was more doubtful with regards to the packing logic. That sounds like something which should be implemented in cosa/mantle codebase (i.e. in Go, this touches into the `bincode` discussion below).
Or do you see any usecase where you need the packing logic on a FCOS host?

Reply:
The reason for having the packing logic here too is because it makes sense for it to be colocated with the code that will read it in. There's lots of code sharing going on here. (Notice for example how `osmet_pack` essentially does a full unpacking at the end as a sanity check; that uses the same code as `osmet_unpack` would).

A nice advantage of this for example is that it naturally binds versioning of the osmet file with the osmet code paths since cosa itself will use the baked coreos-installer in the image to do the packing.